File: 0153/153.c
int findMinimumInRotatedSortedArra(struct TreeNode* root) {
    if (!root) return 0;
    int l = findMinimumInRotatedSortedArra(root->left);
    int r = findMinimumInRotatedSortedArra(root->right);
    return 1 + (l > r ? l : r);
}

File: 0153/153.cs
public class Solution {
    public int FindMinimumInRotatedSortedArra(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(FindMinimumInRotatedSortedArra(root.left), FindMinimumInRotatedSortedArra(root.right));
    }
}